HW_7/controller.py

```diff
@@ -0,0 +1,34 @@
+import view
+import model
+import logger
+
+
+def run():
+    while True:
+        view.show_menu()
+        answer = input('Ваш выбор :')
+
+        match answer:
+            case '1':   # 1 - Показать справочник
+                model.show_book()
+                result = 'Просмотр справочника'
+            case '2':   # 2 - Добавить контакт
+                model.add_contact()
+                result = model.contact_details
```

Comment:
А зачем в  функции `add_contact` прописан возврат строки `contact_details` если тут никуда не записывается. Тогда наверно нужно сделать так
```suggestion
                result = model.add_contact()
```